monitoring/ingestion/ingestion-producer-alert.yaml

```diff
@@ -153,3 +153,18 @@ groups:
       zenko_service: backbeat-ingestion-producer
       description: "Errors in publishing messages to ingestion topic has increased to 5% in the last 5 minutes"
       summary: "Ingestion producer error publishing messages"
+
+- name: Circuit Breaker
+  rules:
+  
+  - alert: CircuitBreakerError
+    Expr: |
+      count(s3_circuit_breaker{namespace="${namespace}", promErr="true"}) by(job) > 0
```

Comment:
this alert definition will match any of the backbeat pods, so no need to duplicate it.
duplication is needed only if we "specialize" each altering rule, e.g. one rule where the expression filters on the job label

Reply:
The alert was kept in lifecycle only